modules/mcuboot/CMakeLists.txt

```diff
@@ -238,6 +238,18 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       return()
     endif()
 
+    # Arguments to imgtool.
+    if(NOT CONFIG_MCUBOOT_EXTRA_IMGTOOL_ARGS STREQUAL "")
```

Comment:
Those extra arguments are also used here:
https://github.com/nrfconnect/sdk-zephyr/blob/f19ebe4b50405b2eb174199b3635f24d741c9bf4/cmake/mcuboot.cmake#L80-L90

I can't determine by the commit message whether or not it has been investigated that the exact same extra arguments can be applied in both places, or whether there could be a risk that arguments specified with the intention of being used in one place could result in a subtle bug at the other location.

Could you please investigate, and update the commit message with a proper description as to why using the same extra arguments in both places are considered safe.